docs-www/src/pages/variables.scss

```diff
@@ -0,0 +1,5 @@
+$colorBHBg: #150940;
+$colorBHMain: #7b59ff;
```

Comment:
IMO, better names would be `colorBhBg` and `colorBhMain` because the company name is Brainhub, not BrainHub